### ISSUE TITLE
Is there anyway to get scan wifi result in other format?

### ISSUE THREAD

**Issue by suijunqiang** [ThanosFisherman/WifiUtils#37]
Hello, 

Is there anyway to get wifi scan result in other format?
Currently your lib return a string but hard to parse into Json or other format, so it's not easy to show the results as ListView on the Android UI.

Thanks & Best Regards
Sui
2019-09-02

**Comment by suijunqiang** [comment]
for more detail the scan results is "Parcelable" list.

**Comment by ThanosFisherman** [comment]
Scan results are returned in a `List<ScanResult>` Just like the normal Android WiFi scan. I don't see any spacial difficulty to parse those into JSON format. What other formats would you recommend?

**Comment by suijunqiang** [comment]
@ThanosFisherman 
First of all, thanks for sharing your amazing project & such quick response.
I commandant to use JSON, would you please give me demo code in your example?

Thanks & Best Regards
Sui

**Comment by ThanosFisherman** [comment]
Have a look at the README file [here](https://github.com/ThanosFisherman/WifiUtils#scanning-for-wifi-networks)

**Comment by suijunqiang** [comment]
@ThanosFisherman 
Yep, I see you have demo in the section of Scanning for WiFi Networks, in fact it has been contained all information that what I need..
How ever, I need some items such as SSID/BSSID/Level and try to show them on the UI, and it seems hard to extract  from ScanResult.

Thanks anyway.
Sui
